pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -22,6 +22,39 @@ const (
 	auditPolicyAsset = "pkg/operator/apiserver/audit/manifests/audit-policies-cm.yaml"
 )
 
+func DefaultPolicy() ([]byte, error) {
+	return getPolicyFromResource("default.yaml")
```

Comment:
`getPolicyFromResource` does not resolve the template varaibles. So we need to use something similar to the beow snippet

```
func DefaultPolicy() ([]byte, error) {
  cm, err := GetAuditPolicies("kube-apiserver-audit-policies", "openshift-kube-apiserver")
  cm.Data["default.yaml"]
}
```

we need to handle error appropriately and a unit test.